src/digitalmarketplace/templates/errors/400.html

```diff
@@ -3,15 +3,17 @@
 {% block pageTitle %}Bad request - Digital Marketplace{% endblock %}
 
 {% block content %}
-  <div class="error-page">
-    <h1 class="govuk-heading-xl">Sorry, there was a problem with your request</h1>
+  <div class="govuk-width-container">
+    <main class="govuk-main-wrapper" role="main">
```

Comment:
Does this need the `main-content` id for the skip link?

Reply:
I'm not sure about adding all this in general, as it means this template can't be used with the base page template from GOV.UK Frontend...